docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +141,364 @@ microservice reports itself as ready.
+.JSON response:
+----
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "LivenessCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566338255331
+      }
+    }
+  ]
+}
+----
+
+Full example code is available link:https://github.com/oracle/helidon/tree/master/examples[here].
+
+
+=== Custom Readiness Health Checks
+
+You can add a readiness check to indicate that the application is ready to be used.  In this
+example, the server will wait five seconds before it becomes ready.
+
+[source,java]
+.Create a new `GreetReadinessCheck` class with the following content:
+----
+@Readiness // <1>
+@ApplicationScoped
+public class GreetReadinessCheck implements HealthCheck {
+  private final AtomicLong readyTime = new AtomicLong(0);
+
+  @Override
+  public HealthCheckResponse call() {
+    return HealthCheckResponse.named("ReadinessCheck")  // <2>
+        .status(isReady())
+        .withData("time", readyTime.get())
+        .build();
+  }
+
+  public void onStartUp(
+      @Observes @Initialized(ApplicationScoped.class) Object init) {
+    readyTime.set(System.currentTimeMillis()); // <3>
+  }
+
+  private boolean isReady() { // <4>
+    return Duration.ofMillis(System.currentTimeMillis() - readyTime.get()).getSeconds() >= 5;
+  }
+}
+----
+<1> Annotation indicating that this is a readiness health check.
+<2> Build the `HealthCheckResponse` with status `UP` after five seconds, else `DOWN`.
+<3> Record the time at startup.
+<4> Become ready after 5 seconds.
+
+[source,bash]
+.Build and run the application.  Issue the curl command with -v within five seconds and you will see that the application is not ready:
+----
+curl -v  http://localhost:8080/health/ready
+----
+
+[source,json]
+.HTTP response:
+----
+...
+< HTTP/1.1 503 Service Unavailable // <1>
+...
+{
+  "status": "DOWN",
+  "checks": [
+    {
+      "name": "ReadinessCheck",
+      "status": "DOWN",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `503` since the application is not ready.
+
+[source,bash]
+.After five seconds you will see the application is ready:
+----
+curl -v http://localhost:8080/health/ready
+----
+
+[source,json]
+.JSON response:
+----
+...
+< HTTP/1.1 200 OK // <1>
+...
+{
+  "status": "UP",
+  "checks": [
+    {
+      "name": "ReadinessCheck",
+      "status": "UP",
+      "data": {
+        "time": 1566399775700
+      }
+    }
+  ]
+}
+----
+<1> The HTTP status is `200` indicating that the application is ready.
+
+Full example code is available link:++https://github.com/oracle/helidon/tree/master/examples[here].
+
+=== Custom Startup Health Checks
+
+You can add a startup check to indicate whether or not the application has initialized to the point that the other health checks make sense.
+In this example, the server will wait eight seconds before it declares itself started.
+
+[source,java]
+.Create a new `GreetStartedCheck` class with the following content:
+----
+@Started // <1>
```

Comment:
The annotation name is `Startup` not started. Probably other naming/text should be aligned

Reply:
Fixed)